eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -3,6 +3,14 @@ steps:
       Write-Host "##vso[task.setvariable variable=VCPKG_BINARY_SOURCES_SECRET;issecret=true;]clear;x-azblob,https://cppvcpkgcache.blob.core.windows.net/public-vcpkg-container,,read"
       Write-Host "##vso[task.setvariable variable=X_VCPKG_ASSET_SOURCES_SECRET;issecret=true;]clear;x-azurl,https://cppvcpkgcache.blob.core.windows.net/public-vcpkg-container,,read"
     displayName: Set Vcpkg Variables
+  
+  - script: vcpkg --version
+    condition: >-
+        and(
+        succeeded(),
+        not(eq(variables['Agent.OS'], 'Darwin'))
```

Comment:
Why are we excluding macos for `vcpkg --version`? Is it not supported there?

Reply:
:D because i'm stupid and i couldn't figure out how to make it work( how to call vcpkg). also it should be the same version on windows/linux so ... not that important IMHO 
time/reward calculation

Reply:
I would have thought the command would be the same in MacOS as Linux (https://the-pi-guy.com/blog/vcpkg_and_macos_building_and_using_packages_on_the_macos_platform/). Maybe it's the vcpkg cli isn't found in the path (or we need to run `./bootstrap-vcpkg.sh` somewhere).

vcpkg is supported on MacOS:
https://learn.microsoft.com/en-us/vcpkg/concepts/supported-hosts#macos

> also it should be the same version on windows/linux so ... not that important IMHO
time/reward calculation

Seems reasonable.

Reply:
i don't want to mess with the environment if i don't have to. besides as mentioned before the version is on linux windows